mozdownload/treeherder.py

```diff
@@ -21,48 +24,77 @@
     'win64': {'build_os': 'win', 'build_architecture': 'x86_64'},
 }
```

Comment:
Something I'm still not clear about is if I should keep this block here or move it into scraper so we have it beside the other constants. It would make it easier for updates, but could confuse because its only relevant for Treeherder. What do you think?